__tests__/app.test.js

```diff
@@ -381,6 +381,38 @@ describe('PATCH /api/reviews/:review_id', ()=>{
     })
 })
 
+describe('GET /api/users',()=>{
+    test('responds with status 200 and an array of objects', ()=>{
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then((res)=>{
+            expect(typeof res.body).toBe('object')
```

Comment:
We don't need to explicitly check for type of object or array here as we are later implicitly proving by using a forEach and objectContaining